src/style.css

```diff
@@ -1 +1,13 @@
-/* styles go here */
+.fieldset {
```

Comment:
Consider using a more descriptive name for the class. Instead of `.fieldset`, you could use `.custom-fieldset` or `.form-fieldset` to indicate its purpose more clearly.